modules/kubernetes-addons/aws-opentelemetry-eks-jmx/locals.tf

```diff
@@ -0,0 +1,69 @@
+locals {
+  default_helm_config = {
+    name             = "opentelemetry"
+    repository       = null
+    chart            = "${path.module}/otel-config"
+    version          = "0.1.0"
+    namespace        = "opentelemetry-operator-system"
+    timeout          = "1200"
+    create_namespace = true
+    description      = "ADOT helm Chart deployment configuration"
+    lint             = false
+    values           = []
+    set              = []
+    set_sensitive    = null
+    postrender       = ""
```

Comment:
these can be removed as we automatically apply the defaults in the `helm-addon` module.